outputs.tf

```diff
@@ -74,6 +74,22 @@ output "cluster_names" {
   ]
 }
 
+output "workload_cluster_id" {
+  description = "The id of the workload cluster"
+  value = flatten([
+    for cluster in ibm_container_vpc_cluster.cluster :
+    cluster.id if cluster.name == "${var.prefix}-workload-cluster"
+  ])
+}
+
+output "management_cluster_id" {
+  description = "The id of the management cluster"
```

Comment:
If these are going to be null or empty if workload or management names were not used, perhaps explain all of this in the output descriptions all way through the final output pattern.

Reply:
Added in the output